src/App.js

```diff
@@ -1,25 +1,8 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return (<Calculator />);
 }
 
```

Comment:
- [ ] Same here, According to the project requirements, The `App.js` component should be a class-based component and not a stateless function component. Your code might look like this 👇🏻 

**./src/App.js**

```javascript
// src/App.js

import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

- Kindly make this 👍🏻